lib/rest/accounts/v1/credential/aws.ts

```diff
@@ -400,85 +411,44 @@ export interface AwsListInstance {
    * If a function is passed as the first argument, it will be used as the callback
    * function.
    *
+   * @param { AwsListInstanceEachOptions } [params] - Options for request
    * @param { function } [callback] - Function to process each record
    */
   each(
     callback?: (item: AwsInstance, done: (err?: Error) => void) => void
   ): void;
-  /**
-   * Streams AwsInstance records from the API.
-   *
-   * This operation lazily loads records as efficiently as possible until the limit
-   * is reached.
-   *
-   * The results are passed into the callback function, so this operation is memory
-   * efficient.
-   *
-   * If a function is passed as the first argument, it will be used as the callback
-   * function.
-   *
-   * @param { AwsListInstanceEachOptions } [params] - Options for request
-   * @param { function } [callback] - Function to process each record
-   */
   each(
-    params?: AwsListInstanceEachOptions,
+    params: AwsListInstanceEachOptions,
     callback?: (item: AwsInstance, done: (err?: Error) => void) => void
   ): void;
-  each(params?: any, callback?: any): void;
-  /**
-   * Retrieve a single target page of AwsInstance records from the API.
-   *
-   * The request is executed immediately.
-   *
-   * If a function is passed as the first argument, it will be used as the callback
-   * function.
-   *
-   * @param { function } [callback] - Callback to handle list of records
-   */
-  getPage(
-    callback?: (error: Error | null, items: AwsPage) => any
-  ): Promise<AwsPage>;
```

Comment:
Should this have been removed? Can no longer just pass in a callback.

Reply:
getPage implementation called [BaseTwilio.ts request](https://github.com/twilio/twilio-node/blob/4.0.0-rc/lib/base/BaseTwilio.ts#L121) and an error will be thrown if user does not pass in `targetURL`. I think that user should not be able to call getPage with just a callback - just to get an error later